### DISCUSSION TITLE
Bench type / sensor requirements

### DISCUSSION THREAD

**Discussion by stefan63** [DIY-Flow-Bench/DIY-Flow-Bench#176] [Shield Design]
**Starting this thread with a statment and are asking for confirmation or correction in my assumptions.**
I plan to bild a MAF based bench and I assume from my examination of the code and reading true the documentation that a bord with a BME280+MAF sensor + 1x diff pressure sensors is all I nead.  Iff I also want to add pitot sensor I will ned a 2nd diff press sensor. 
Did I get this right?

**Comment by DeeEmm** [comment]
Yes. this is correct.

**MAF** - For Mass flow measurements
**pRef** - Reference pressure
**BME** - Environmental data 

pRef and BME are required to convert mass flow to volumetric flow. Alternatively you can used fixed values in the code and just use the MAF sensor, however this will obviously generate inaccurate cfm readings.

Optional..

**pDiff** is required for differential style benches but you can also use / record this value if you want to measure pressure / vac at a fixed point on your system
**pitot** for a pitot probe if you wish to use one

**Reply by DeeEmm** [reply]
Oh also, by the way.

I will have PCBs available to purchase soon. Just need to validate them. Unfortunately been tied up with real-life stuff.

/DM
